staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -70,8 +70,9 @@ func (t *azureDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Vol
 	azureSource := volume.AzureDisk
 	pv := &v1.PersistentVolume{
 		ObjectMeta: metav1.ObjectMeta{
-			// A.K.A InnerVolumeSpecName required to match for Unmount
-			Name: volume.Name,
+			// Must be unique per disk as it is used as the unique part of the
+			// staging path
+			Name: azureSource.DiskName,
```

Comment:
@andyzhangx is it ok if I just use the disk name from here?

Reply:
lgtm